src/modules/Sidebar/Sidebar.js

```diff
@@ -40,6 +43,9 @@ class Sidebar extends Component {
     /** Direction the sidebar should appear on. */
     direction: PropTypes.oneOf(['top', 'right', 'bottom', 'left']),
 
+    /** Called when the user clicks away from the sidebar. */
+    onSidebarBlur: PropTypes.func,
```

Comment:
Since this is a prop on the Sidebar component, let's leave out the word `Sidebar` in the prop name:

```jsx
<Sidebar onBlur={} />
```